packages/phone-number-privacy/signer/src/signing/get-partial-signature.ts

```diff
@@ -186,7 +192,7 @@ export async function handleGetBlindedMessagePartialSig(
     }
     Counters.responses.labels(Endpoints.GET_BLINDED_MESSAGE_PARTIAL_SIG, '200').inc()
     logger.info({ response: signMessageResponse }, 'Signature retrieval success')
-    response.json(signMessageResponse)
+    response.set('keyVersion', key.version.toString()).json(signMessageResponse)
```

Comment:
Related to suggestions elsewhere
```suggestion
    response.set('Odis-Key-Version', key.version.toString()).json(signMessageResponse)
```